db/migrate/20170822144440_add_first_name_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddFirstNameToUser < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :first_name, :string
```

Comment:
This may be tricky to undo and then redo. Do your best to have a look at how to do this